src/ProjectService.Models.Broker/Requests/IGetUserDataRequest.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace LT.DigitalOffice.Broker.Requests
 {
-    [AutoInjectRequest(nameof(RabbitMqConfig.GetUserDataEndpoint))]
+    [AutoInjectRequest(nameof(RabbitMqConfig.FindDepartmentsEndpoint))]
     public interface IGetUserDataRequest
```

Comment:
why you dont rename interface?

Reply:
It is my mistake, I don't know why I changed it, I returned it as it was